homeassistant/components/comfoconnect/sensor.py

```diff
@@ -72,163 +76,187 @@
+        device_class=None,
+        name="Supply fan duty",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:fan-plus",
+        sensor_id=SENSOR_FAN_SUPPLY_DUTY,
+    ),
+    ATTR_EXHAUST_FAN_SPEED: ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_FAN_SPEED,
+        device_class=None,
+        name="Exhaust fan speed",
+        unit_of_measurement="rpm",
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_SPEED,
+    ),
+    ATTR_EXHAUST_FAN_DUTY: ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_FAN_DUTY,
+        device_class=None,
+        name="Exhaust fan duty",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_DUTY,
+    ),
+    ATTR_EXHAUST_TEMPERATURE: ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_TEMPERATURE,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        name="Exhaust temperature",
+        unit_of_measurement=TEMP_CELSIUS,
+        sensor_id=SENSOR_TEMPERATURE_EXHAUST,
+        multiplier=0.1,
+    ),
+    ATTR_EXHAUST_HUMIDITY: ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_HUMIDITY,
+        device_class=DEVICE_CLASS_HUMIDITY,
+        name="Exhaust humidity",
+        unit_of_measurement=PERCENTAGE,
+        sensor_id=SENSOR_HUMIDITY_EXHAUST,
+    ),
+    ATTR_AIR_FLOW_SUPPLY: ComfoconnectSensorEntityDescription(
+        key=ATTR_AIR_FLOW_SUPPLY,
+        device_class=None,
+        name="Supply airflow",
+        unit_of_measurement=VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
+        icon="mdi:fan-plus",
+        sensor_id=SENSOR_FAN_SUPPLY_FLOW,
+    ),
+    ATTR_AIR_FLOW_EXHAUST: ComfoconnectSensorEntityDescription(
+        key=ATTR_AIR_FLOW_EXHAUST,
+        device_class=None,
+        name="Exhaust airflow",
+        unit_of_measurement=VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_FLOW,
+    ),
+    ATTR_BYPASS_STATE: ComfoconnectSensorEntityDescription(
+        key=ATTR_BYPASS_STATE,
+        device_class=None,
+        name="Bypass state",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:camera-iris",
+        sensor_id=SENSOR_BYPASS_STATE,
+    ),
+    ATTR_DAYS_TO_REPLACE_FILTER: ComfoconnectSensorEntityDescription(
+        key=ATTR_DAYS_TO_REPLACE_FILTER,
+        device_class=None,
+        name="Days to replace filter",
+        unit_of_measurement=TIME_DAYS,
+        icon="mdi:calendar",
+        sensor_id=SENSOR_DAYS_TO_REPLACE_FILTER,
+    ),
+    ATTR_POWER_CURRENT: ComfoconnectSensorEntityDescription(
+        key=ATTR_POWER_CURRENT,
+        device_class=DEVICE_CLASS_POWER,
+        name="Power usage",
+        unit_of_measurement=POWER_WATT,
+        sensor_id=SENSOR_POWER_CURRENT,
+    ),
+    ATTR_POWER_TOTAL: ComfoconnectSensorEntityDescription(
+        key=ATTR_POWER_TOTAL,
+        device_class=DEVICE_CLASS_ENERGY,
+        name="Power total",
```

Comment:
This should be "Energy total"

https://www.home-assistant.io/docs/energy/faq/#energy-vs-power

Reply:
Watt = power, Wh = energy.